src/main/java/dev/blachut/svelte/lang/parsing/html/SvelteHtmlLexer.kt

```diff
@@ -1,15 +1,42 @@
 package dev.blachut.svelte.lang.parsing.html
 
 import com.intellij.lang.HtmlScriptContentProvider
+import com.intellij.lang.Language
 import com.intellij.lang.javascript.JSStubElementTypes
 import com.intellij.lang.javascript.dialects.JSLanguageLevel
 import com.intellij.lang.javascript.types.JSEmbeddedContentElementType
-import com.intellij.lexer.*
+import com.intellij.lexer.HtmlHighlightingLexer
```

Comment:
Sorry for this, automatic refactor. Can undo this

Reply:
Don't worry, it's fine